.github/workflows/integration.yml

```diff
@@ -0,0 +1,39 @@
+name: .NET Integration Test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 5.0.x
+      - name: Restore Dependencies
+        run: cd src && dotnet restore
```

Comment:
We made this adjustment in another library, but how about doing this and passing the directory instead of using `cd`?

`run dotnet restore src`

This comment applies here and for the other `run` steps below.